src/main/java/com/aws/greengrass/shadowmanager/ipc/UpdateThingShadowRequestHandler.java

```diff
@@ -129,6 +131,14 @@ public UpdateThingShadowHandlerResponse handleRequest(UpdateThingShadowRequest r
                             .orElseThrow(() ->
                                     new InvalidRequestParametersException(ErrorMessage
                                             .createInvalidPayloadJsonMessage("")));
+
```

Comment:
nit: Maybe we should add a TODO here to throw an exception when we want to move to a major version